traad/app.py

```diff
@@ -181,10 +181,21 @@ def get_imports_view(context):
 @standard_refactoring
 def rename_view(context):
     args = bottle.request.json
+
+    log.info("`rename_view` called: {}".format(args))
+
+    # TODO 1: Modify "emacs-traad" to use these parameters.
+    docs = bool(args.get('docs', False))
+    unsure = bool(args.get('unsure', False))
```

Comment:
You originally set `unsure` to `None` when it wasn't in `args`, but now you set it to `False`. Looking at the docstring for`workspace.rename`, it looks like it's expecting `None`. Do we need to change this code, or is it ok the way it is?

Reply:
That's an error on my part. I'll fix it.